pkg/autoscaler/metrics/wire.proto

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package metrics;
+
+message WireKey {
+    string namespace = 1;
+    string name = 2;
+}
+
+message WireStat {
+  string podName = 1;
```

Comment:
Do we foresee a case where we might want to provide information about a a container? Or put another way, should there be a container field as well?

Reply:
I don't think so, this is purely a mirror of the "Stat" structs with the unnecessary fields stripped off of it. It's also purely for use between the autoscaler and the queue-proxy/activator, neither of which currently report any container specific data. It's all data we fully control.

Reply:
Yeah, the one (obvs not baked thought) is that with the sidecars, there's been talk about supporting pull based systems, where containers might have different information, and hence it might be necessary. 
But looks great to me 👍

Reply:
sure I think this'll be trivial to extend later on